OpenSim/Analyses/StaticOptimizationTarget.cpp

```diff
@@ -586,9 +583,11 @@ computeConstraintVector(SimTK::State& s, const Vector &parameters,Vector &constr
     Vector actualAcceleration(getNumConstraints());
     computeAcceleration(s, parameters, actualAcceleration);
 
+    auto coordinates = _model->getCoordinatesInMultibodyTreeOrder();
```

Comment:
Good point. Let's compare:
Current PR `testStaticOptimization` on Appveyor x86: [114.84](https://ci.appveyor.com/project/opensim-org/opensim-core/build/4921/job/w95h8obwmfecua0t#L596)
PR #1279  `testStaticOptimization` on Appveyor x86: [112.95] (https://ci.appveyor.com/project/opensim-org/opensim-core/build/4918/job/5cbpa88fnqra9cox#L530)

The overhead of rummaging through coordinates appears to be small compared to performing the optimization at each step. The `computeConstraintVector` is called prior to optimization in `prepareToOptimize`.